build/shared/lib/languages/PDE.properties

```diff
@@ -445,6 +445,18 @@ ensure_exist.messages.unrecoverable.description = Could not properly re-save the
 # Check name
 check_name.messages.is_name_modified = The sketch name had to be modified. Sketch names can only consist\nof ASCII characters and numbers (but cannot start with a number).\nThey should also be less than 64 characters long.
 
+# External changes detector
+change_detect.reload.title=Tab modified externally
+change_detect.reload.question="%s" was modified by another program.
+change_detect.reload.comment=Would you like to keep this version or load the new changes?\nEither way, the version you discard will be saved to your sketch folder.
```

Comment:
The language needs to be rewritten in the first, but it's mostly in the direction we need to go. I think dropping the final sentence would get rid of most of the awkwardness. It could even be shown in a second dialog box (or just in the console as a message). 

Your suggested version works great for people already familiar with IDEs or advanced text editors that have this kind of feature. For the majority of our audience, it's not clear what "reload" really means in this context: do I lose my work? Does it save something? What's happening?